tests/wpunit/Tribe/Tickets/Tickets_Sale_End_DateTest.php

```
@@ -0,0 +1,158 @@
+				'ticket_start_time'       => '08:00:00',
+				'ticket_end_date'         => '2026-09-15',
+				'ticket_end_time'         => '08:00:00',
+			],
+			$overrides
+		);
+
+		$ticket_id = $rsvp->ticket_add( $post_id, $data );
+
+		$this->assertNotEmpty( $ticket_id );
+
+		return $ticket_id;
```

PHPStan (2.2.8)

[error] Method Tribe\Tickets\Tickets_Sale_End_DateTest::create_ticket() should return int but returns bool.


(return.type)